lib/_stream_transform.js

```diff
@@ -67,12 +67,26 @@ const { Object } = primordials;
 
 module.exports = Transform;
 const {
+  ERR_INVALID_ARG_TYPE,
   ERR_METHOD_NOT_IMPLEMENTED,
   ERR_MULTIPLE_CALLBACK,
   ERR_TRANSFORM_ALREADY_TRANSFORMING,
   ERR_TRANSFORM_WITH_LENGTH_0
 } = require('internal/errors').codes;
 const Duplex = require('_stream_duplex');
+// eslint-disable-next-line func-style
+const { constructor: AsyncGeneratorFunction } = async function * () {};
+const AsyncIteratorPrototype = Object.getPrototypeOf(
+  Object.getPrototypeOf(async function* () {}).prototype);
+
+const kSourceIteratorPull = Symbol('SourceIteratorPull');
+const kSourceIteratorResolve = Symbol('SourceIteratorResolve');
+const kSourceIteratorChunk = Symbol('SourceIteratorChunk');
+const kSourceIteratorStream = Symbol('SourceIteratorStream');
+const kSourceIteratorPump = Symbol('SourceIteratorPump');
+const kSourceIteratorGenerator = Symbol('SourceIteratorGenerator');
+const kSourceIteratorGrabResolve = Symbol('SourceIteratorGrabResolve');
```

Comment:
Suggestion: Follow the [guidelines](https://github.com/nodejs/node/blob/master/doc/guides/using-symbols.md) and use identifiers which match the symbol descriptions for easier code searching.

Reply:
so we're saying that the symbol description strings should also be prefixed with "k"?

Reply:
Yes, exactly